res/database/Default/database_scripts.xml

```diff
@@ -572,36 +575,42 @@
 				<scripttemplate index="0" product="2" licence_type="2" guid="scripttemplate-random-ron-animalseries01_01">
 					<title>
 						<de>%TITLEEPISODE1%</de>
+						<en>%TITLEEPISODE1%</en>
```

Comment:
Is this where you noticed the variable resolution problem?

Reply:
possibly.
But definitely I remember `scripttemplate-ron-musiksterneamabend`

Reply:
fix proposed, but change does not hurt either.